discovery-infra/start_discovery.py

```diff
@@ -395,10 +507,7 @@ def main():
         "--vm-network-cidr",
         help="Vm network cidr",
         type=str,
-        default="192.168.126.0/24",
-    )
-    parser.add_argument(
-        "-nN", "--network-name", help="Network name", type=str, default="test-infra-net"
+        required=False,
```

Comment:
u don't need to set required=False it is default option. 
all params that has -- are optional

Reply:
don't know why I did that